### ISSUE TITLE
Screenreader: Ignore Bullet Points and Icons

### ISSUE THREAD

**Issue by fabianbrandscheid** [flutter/flutter#154500]
### Use case

We use the `MarkdownBody` Widget to show some intros for our App. The intro texts are stored in a .json file.

We carried out a professional accessibility test and one of the findings we noticed was that the intro text is read out in an unusual way. 
Specifically, it was noted that icons are read aloud and that the bullet point is read aloud each time a list is given.
It would be desirable if the `MarkdownBody` widget offered a configuration option to influence the behavior of the screen reader.

In my opinion, an option to not read out icons when the screen reader is switched on would be useful.
In addition, not to explicitly read out the bullet points for enumerations.
Currently, the behavior of the screen reader cannot be influenced here

### Proposal

Example Text:
```
🚀 **Improvements**
* Feature A
* Feature B
```
--> The Screenreader read the following:
_Rocket_, Improvments
_Dot_, Feature A
_Dot_ Feature B

The accessibility testers noted that reading out the term “rocket” does not offer any added value here, as it is only a visual element. These are not usually read aloud.

The same applies to reading out “dot”. It is implicitly clear that this is an enumeration and reading this out again is confusing.

**Comment by chunhtai** [comment]
I think the package can strip out the specially character from the semantic label, and no further support form framework is needed

**Comment by fabianbrandscheid** [comment]
@chunhtai Do you may have an example?
When I wrap my `MarkdownBody` with a Semantic Widget it still not working.

Thanks in advice !

**Comment by fabianbrandscheid** [comment]
Any news on this topic?
